app/grants/models/grant.py

```diff
@@ -817,6 +856,10 @@ def repr(self, user, build_absolute_uri):
 
         grant_tags = serializers.serialize('json', self.tags.all(),fields=['id', 'name'])
 
+        active_round_names= []
+        for active_round in self.in_active_clrs.all():
+            active_round_names.append(active_round.display_text)
+
```

Comment:
would be more performant if we do

```active_round_names = self.in_active_clrs.values_list('display_text', flat=True)```

Reply:
hmm this won't work as such ! I might have to typcast it into a list